dev-docs/source/Testing/SANSGUI/SANSGUITests.rst

```diff
@@ -51,14 +51,18 @@ Data reduction
     - Check the ``Multi-period`` box
```

Comment:
I don't think the `Export Table` is saving out the multi-period settings, I can't see the 6 additional columns in the exported CSV file, and loading it back removes the `Multi-period` check, and any data from the columns.

The instructions make it seem like the `Multi-period` fields should be loaded back

Reply:
I agree it's not clear. I've moved the lines relating to `Export Table` to earlier in the script, which should remove ambiguity